docs/topics/media-pipeline.rst

```diff
@@ -243,26 +243,21 @@ Usage example
 .. setting:: IMAGES_URLS_FIELD
 .. setting:: IMAGES_RESULT_FIELD
 
-In order to use a media pipeline first, :ref:`enable it
+In order to use a media pipeline, first :ref:`enable it
 <topics-media-pipeline-enabling>`.
 
-Then, if a spider returns a dict with the URLs key (``file_urls`` or
-``image_urls``, for the Files or Images Pipeline respectively), the pipeline will
-put the results under respective key (``files`` or ``images``).
+Then, if a spider returns an :ref:`item object <topics-items>` with the URLs
+field (``file_urls`` or ``image_urls``, for the Files or Images Pipeline
+respectively), the pipeline will put the results under respective field
+(``files`` or ``images``).
 
-If you prefer to use :class:`~.Item`, then define a custom item with the
-necessary fields, like in this example for Images Pipeline::
-
-    import scrapy
-
-    class MyItem(scrapy.Item):
-
-        # ... other item fields ...
-        image_urls = scrapy.Field()
-        images = scrapy.Field()
+When using :ref:`item types <item-types>` where fields are defined beforehand,
+you must define both the URLs field and the results field. For example, when
+using the images pipeline, items must define both the ``image_urls`` and the
+``images`` field.
 
 If you want to use another field name for the URLs key or for the results key,
-it is also possible to override it.
+it is also possible to override it:
```

Comment:
:lipstick: No strong opinion (and there’s a high chance I’m the author of this line change), but I think it may make sense, given this colon, to convert the two items below into unordered list items.

Reply:
In that case, let's revert it. The PR is long enough as it is, and I don't think this specific change is necessary for the overall PR to be understood.